components/tests/ui/robot_setup.sh

```diff
@@ -45,6 +47,34 @@ echo "A2,Treatment,10" >> "$BULK_ANNOTATION_CSV"
 # Create file for upload as File Annotation
 echo "Robot test file annotations" > "$FILE_ANNOTATION"
 
```

Comment:
If I call ```conn.close()``` at the end of wsScript.py then this kills the session that I have joined and the rest of the setup script fails.
The wsScript.py doesn't create any stateful services and the whole session is completed at the end of the setup script with ```$ bin/omero logout``` so I don't think we need to close the BlitzGateway in wsScript.py.